src/web_interface/static/js/system_health.js

```diff
@@ -32,8 +32,19 @@ async function updateSystemHealth() {
             const queueElement = document.getElementById("backend-unpacking-queue");
             if (entry.unpacking.unpacking_queue > 500) {
                 queueElement.classList.add("text-warning");
+            } else {
+                queueElement.classList.remove("text-warning");
             }
             queueElement.innerText = entry.unpacking.unpacking_queue.toString();
+
+            const throttleElement = document.getElementById("backend-unpacking-throttle-indicator");
+            if (entry.unpacking.is_throttled) {
+                throttleElement.innerText = '<i class="fa-regular fa-circle-pause"></i>';
```

Comment:
this should be `innerHTML` and "fa-regular fa-circle-pause" won't render for me (do we have an old version of font awesome?)
```suggestion
                throttleElement.innerHTML = '<i class="far fa-pause-circle"></i>';
```